packages/syft/src/syft/service/attestation/attestation_service.py

```diff
@@ -0,0 +1,65 @@
+# stdlib
+from collections.abc import Callable
+
+# third party
+import requests
+
+# relative
+from ...serde.serializable import serializable
+from ...store.document_store import DocumentStore
+from ..context import AuthedServiceContext
+from ..response import SyftError
+from ..response import SyftSuccess
+from ..service import AbstractService
+from ..service import service_method
+from ..user.user_roles import GUEST_ROLE_LEVEL
+from .attestation_constants import ATTESTATION_SERVICE_URL
+from .attestation_constants import ATTEST_CPU_ENDPOINT
+from .attestation_constants import ATTEST_GPU_ENDPOINT
+
+
+@serializable()
+class AttestationService(AbstractService):
+    """This service is responsible for getting all sorts of attestations for any client."""
+
+    def __init__(self, store: DocumentStore) -> None:
+        self.store = store
+
+    def perform_request(
+        self, method: Callable, endpoint: str, raw: bool = False
+    ) -> SyftSuccess | SyftError | str:
+        try:
+            response = method(f"{ATTESTATION_SERVICE_URL}{endpoint}")
+            response.raise_for_status()
+            message = response.json().get("result")
+            raw_token = response.json().get("token")
+            if raw:
+                return raw_token
+            elif message:
```

Comment:
Since we return a string as true or false, we could use the utility function called str_to_bool to make it such that  we convert it to a boolean value.

Reply:
Resolved by [249e309](https://github.com/OpenMined/PySyft/pull/8659/commits/249e309cf92d0d9c32cba0dd5b4fcdbdb71f3a1b)

Reply:
Also Tested it on Cloud on the same.